site/content/docs/developer/building-zap-with-intellij-idea.md

```diff
@@ -13,6 +13,19 @@ This guide explains how to make changes to ZAP using IntelliJ IDEA.
 ## Preparation
 You will need to have followed the [Quick Start Guide to Building ZAP](../quick-start-build/) and installed a version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/).
 
+##### Gradle Resources
+Working with ZAP in IntelliJ IDEA may need a bit more Java resources for the gradle actions. To adjust how much memory gradle can use on your machine set the options in [gradle.properties](https://docs.gradle.org/current/userguide/build_environment.html)
```

Comment:
- `gradle` → `Gradle`
 - `options` → `option`

Reply:
updated and ready for review